### PR TITLE
EB-3071-branch : resolved github history mismatch

### PR DESCRIPTION
**Pull request by DuncanRN**

### CODE REVIEW

File: edgebuilder-install.sh [hunk 1]
```diff
@@ -468,28 +468,18 @@ install_cli_rpm()
 uninstall_server()
 {
     export DEBIAN_FRONTEND=noninteractive
+    # check if edgebuilder-server is currently installed
```
Comment (rbose21) [inline edgebuilder-install.sh:471]: we need to first bring down the server components and all our containers and images before uninstalling the packages

File: edgebuilder-install.sh [hunk 2]
```diff
@@ -468,28 +468,18 @@ install_cli_rpm()
...
     if dpkg -s edgebuilder-server; then
 
-        sudo rm -rf /opt/edgebuilder/server/vault
-        # attempt autoremove
-        if sudo apt autoremove -qq edgebuilder-server -y ;then
-            echo "Successfully autoremoved server components"
+        sudo rm -rf /opt/edgebuilder/server
```
Comment (rbose21) [inline edgebuilder-install.sh:474]: this should be /opt/edgebuilder we we dont expect the server and node to be installed together

File: edgebuilder-install.sh [hunk 3]
```diff
@@ -468,28 +468,18 @@ install_cli_rpm()
...
+        sudo apt-get -qq remove edgebuilder-server -y
+        if (dpkg --list edgebuilder-server |grep "^rc") || !(dpkg --list edgebuilder-server); then
```
Comment (rbose21) [inline edgebuilder-install.sh:476]: I dont think the rc check is required, could you please confirm after making the above updates?

File: edgebuilder-install.sh [hunk 4]
```diff
@@ -500,64 +490,43 @@ uninstall_server()
...
-  else
-      # package not currently installed, so exit
-      echo "edgebuilder-node NOT currently installed"
-      exit 0
-  fi
+   # check if edgebuilder-node is currently installed
+   if dpkg -s edgebuilder-node; then
+       sudo apt-get -qq remove edgebuilder-node -y
+
+       if (dpkg --list edgebuilder-node |grep "^rc") || !(dpkg --list edgebuilder-node); then
```
Comment (rbose21) [inline edgebuilder-install.sh:497]: same comment as before

File: edgebuilder-install.sh [hunk 5]
```diff
@@ -500,64 +491,44 @@ uninstall_server()
...
 uninstall_node()
 {
-  if dpkg -s edgebuilder-node; then
-
-      # attempt autoremove
-      if sudo apt autoremove -qq edgebuilder-node -y ;then
-          echo "Successfully autoremoved node components"
+   # check if edgebuilder-node is currently installed
+   if dpkg -s edgebuilder-node; then
+      sudo edgebuilder-node down -v
```
Comment (rbose21) [inline edgebuilder-install.sh:496]: I dont think there's a -v option for edgebuilder-node script as it sets the -v option internally when it calls docker-compose

File: edgebuilder-install.sh [hunk 6]
```diff
@@ -1,10 +1,8 @@
 #!/bin/sh
-LOGFILE=eb-installer.log
```
Comment (TomFlem) [inline edgebuilder-install.sh:2]: Why change this?

File: edgebuilder-install.sh [hunk 7]
```diff
@@ -1,10 +1,8 @@
...
+
 log() {
     echo "[$(date +"%T-%D")]" "$@"
 }
-exec 3>&1 1>"$LOGFILE" 2>&1
```
Comment (TomFlem) [inline edgebuilder-install.sh:6]: Put this back please

File: edgebuilder-install.sh [hunk 8]
```diff
@@ -1,10 +1,8 @@
...
-set -x
```
Comment (TomFlem) [inline edgebuilder-install.sh:7]: Put this back please

File: edgebuilder-install.sh [hunk 9]
```diff
@@ -570,9 +549,11 @@ display_usage()
   echo "     -u, --uninstall          : Uninstall the package"
   echo "     -f, --file               : Absolute path to local package"
   echo "     --offline-provision      : Enable offline node provision"
+  echo "     --install-docker         : Install docker as part of package install"
```
Comment (TomFlem) [inline edgebuilder-install.sh:552]: Add >&3 to the end of each line to output

File: edgebuilder-install.sh [hunk 10]
```diff
@@ -610,17 +595,21 @@ while [ "$1" != "" ]; do
...
 
 # If not run as sudo, exit
-if [ "$(id -u)" -ne 0 ]
-  then log "Insufficient permissions, please run as root/sudo" >&3
+if [ "$(id -u)" -ne 0 ]; then
+  echo "Insufficient permissions, please run as root/sudo"
   exit 1
 fi
 
+exec 3>&1 1>"$LOGFILE" 2>&1
```
Comment (TomFlem) [inline edgebuilder-install.sh:611]: Put back please

**Review by TomFlem — CHANGES_REQUESTED** [review]

File: edgebuilder-install.sh (hunk 7 above)
Comment (rbose21) [reply edgebuilder-install.sh:6]: done

File: edgebuilder-install.sh (hunk 8 above)
Comment (rbose21) [reply edgebuilder-install.sh:7]: done

File: edgebuilder-install.sh (hunk 6 above)
Comment (rbose21) [reply edgebuilder-install.sh:2]: reinstated

File: edgebuilder-install.sh (hunk 9 above)
Comment (rbose21) [reply edgebuilder-install.sh:552]: done

File: edgebuilder-install.sh (hunk 10 above)
Comment (rbose21) [reply edgebuilder-install.sh:611]: done

**Review by TomFlem — APPROVED** [review]
LGTM

**Pull request merged**